src/database/sql/bundles/import.sql

```diff
@@ -89,11 +89,13 @@ UPDATE SET
 INSERT INTO new_data_items (
   id, parent_id, root_transaction_id, height, signature, anchor,
   owner_address, target, data_offset, data_size, content_type,
-  tag_count, indexed_at, signature_type
+  tag_count, indexed_at, signature_type, offset, size, owner_offset,
+  owner_size, signature_offset, signature_size
```

Comment:
we use the `size` namespace quite a lot here (`data_size`, `owner_size`, etc) and now adding `size` itself. I understand its the total size of the data item with its headers. wondering if theres opportunity here for describing that in the column

Reply:
@djwhitt what do you think about this?

Reply:
It breaks symmetry with `offset` if we only add a prefix to `size`, but we could consider `raw_offset` and `raw_size`.